packages/truffle-core/cli.js

```diff
@@ -47,7 +47,9 @@ command.run(process.argv.slice(2), options, function(err) {
     if (typeof handle.close === 'function'){
       handle.close();
     } else if (handle.readable && !handle._isStdio){
-      handle.destroy();
+      //This is causing problems over RPC
+      //with hd-wallet-provider V5
+      //handle.destroy();
```

Comment:
NB: this was put in because there was crashing when `config.websockets === true` in `darq-truffle`. Needs further investigation.

Reply:
Maybe I've just been overly-scolded in the past for even the tiniest amounts of commented-code, but now I feel the need to pass on this sentiment.

Idea: instead of commenting out code, we make sure to remove it via PR with a reference to the upstream issue, so that way GH make it easy for us when the issue closes, and we can add it back in with the revert button.

Reply:
Oh totally that's a good idea. . . This comment is really for me - like DO NOT forget this is a serious problem right now. Needs resolution.